src/util/fs.js

```diff
@@ -450,6 +450,8 @@ async function buildActionsForHardlink(
 
       // push all files to queue
       invariant(srcFiles, 'src files not initialised');
+      // hardlinking is per package, not going into sub packages
+      srcFiles = srcFiles.filter(f => f !== 'node_modules');
```

Comment:
Why is there paths from the node_modules here? Shouldn't they be handled outside of this function (not completely sure how works the linking pipeline yet)? + will it work with `bundledDependencies`?